onnxruntime/core/providers/rocm/roctracer_manager.cc

```diff
@@ -0,0 +1,465 @@
+#include <time.h>
+#include <cstring>
+#include <chrono>
+
+#include "roctracer_manager.h"
+
+namespace onnxruntime {
+namespace profiling {
+
+// allocate a 16K buffer for recording async activities
+static constexpr size_t sc_activity_buffer_size = 0x4000;
```

Comment:
nit: we are (loosely) following google c++ guideline, so this should be kBlahBlah...

https://google.github.io/styleguide/cppguide.html#Constant_Names